sirepo/package_data/static/js/raydata.js

```diff
@@ -809,15 +828,39 @@ SIREPO.app.directive('scansTable', function() {
             };
 
             $scope.columnIsSortable = (column) => {
-                return column !== 'stop';
+                if (['stop'].includes(column)) {
```

Comment:
return false/ true from an if/else statement is a code smell. Almost always you can instead return the result of the test.

` return ! (column === 'stop' || $scope.scans.length > 0 && $scope.scans[0][column] !== null && typeof $scope.scans[0][column] === 'object');`

Reply:
I honestly preferred the way I had it, it was easier to read and understand the different cases, and therefore probably easier to change in the future. [7322d19](https://github.com/radiasoft/sirepo/pull/7163/commits/7322d198d8db767467b2dfee55a74b6b9b0cba27)

Reply:
It is an interesting matter of style. Our style tends to favor less code over more in general. Fewer locals, omitting else, etc. I would put this in that category. I find following the braces harder to read.

Some linters complain about code with boolean tests that return the result of the test
https://pylint.readthedocs.io/en/stable/user_guide/messages/refactor/simplifiable-if-expression.html